src/utils/consts.tsx

```diff
@@ -2,8 +2,7 @@ import { Label } from '@patternfly/react-core/dist/dynamic/components/Label';
 import React from 'react';
 
 // for the support cases widget
-export const url =
-  'https://api.access.stage.redhat.com/support/v1/cases/filter';
+export const url = 'https://api.access.redhat.com/support/v1/cases/filter';
```

Comment:
is it supposed to use this url even in stage environment?

Reply:
yes, I was only using this to test because I could only open a test case in stage. No one should open a support case in stage(technically).